core/templates/pages/story-editor-page/editor-tab/story-editor.component.html

```diff
@@ -235,6 +239,126 @@ <h3 class="story-card-header">Story Card</h3>
         </div>
       </mat-card>
     </div>
+
+    <div class="chapter-list-container" *ngIf="platformFeatureService.status.SerialChapterLaunchCurriculumAdminView.isEnabled">
```

Comment:
Make a function in corresponding `.ts` file to get the status of the feature from platformFeatureService and then use it here.

Reply:
Done